Gemfile

```diff
@@ -17,11 +17,14 @@ group :development do
   gem 'rspec'
   gem 'shotgun'
   gem 'solargraph'
+  gem 'debase'
+  gem 'ruby-debug-ide', require: false
 end
 
 group :test do
   gem 'rack-test', require: 'rack/test'
   gem 'simplecov', require: false
+  gem 'json-schema'
```

Comment:
where is this gem actually being used?

Reply:
nowhere, really, was planning on using it for testing. I'll remove it.